src/brasil/gov/portal/browser/viewlets/templates/logo.pt

```diff
@@ -6,9 +6,9 @@
                        title view/description">
             <span id="portal-title-1"
                   tal:content="view/title_1">Secretaria de</span>
-            <h1 id="portal-title"
+            <div id="portal-title"
```

Comment:
IMO, this change could cause problems on third-party themes.

Reply:
Unfortunately, this has to be done or we will have accessibility errors in ACCESS tool. We'll add an entry to changelog.